Include financial updates (#877)
publiccodenet/about: activities/organization/finance-updates/finance-update-december-2020.md

> ---

Reviewer: This folder structure is incorrect: `` activities/organization/finance-updates/``
It creates a new ``organization`` folder within ``activities``.
It should be ``/organization/finance-updates/``.

Author: and this is what threw me off, I'll correct it

type: resource
---

# Finance update December 2020

## Summary

> * Current cash at bank position: € 1.370.316,45

Reviewer: In American and British English, it's correct to represent this number as €1,370,316.45. Would it be difficult to reverse all commas and decimals from the Dutch financial tracking tools for these reports? It would bring the reports in line with the English used on the rest of our website.

Author: It's correct in the financial tools. These one-pagers are made manually, and thus the error is on my side. Thank you for the diligence <3

* Cost incurred (cash burned):  € 344k (2019) + € 758k (till Nov 2020) = € 1.102.000
* Projected runway:  1.370.316 / 72.000 = 19 months (current cash position / average expenses upcoming 12 months): 17,9 months from January 2021

## Funding

> The Foundation started in June 2019 with a startup-loan of € 1 million (€ 0,2mio + 0,8mio). This loan (including accrued interest € 29.691) is paid back on the 24th August 2020. On the 24th August the Foundation received a loan of € 1.702.200 that is part of a fiscal gift schema. Furthermore the first (formal fiscal) gift was received € 435.500.  In total a net-amount of € 1.137.700 (€ 1,7mio + 0,4mio -/- 1mio) was received.

Reviewer: ```suggestion
The Foundation started in June 2019 with a startup-loan of € 1 million (€ 0,2mil + 0,8mil). This loan (including accrued interest € 29.691) is paid back on the 24 August 2020. On 24 August the Foundation received a loan of € 1.702.200 that is part of a fiscal gift scheme. Furthermore the first (formal fiscal) gift of € 435.500 was received.  In total, a net amount of € 1.137.700 (€ 1,7mil + 0,4mil -/- 1mil) was received.
```

> In October a decision was made to change the structure of the gift. On November 1st a transfer of € 1.726.000 was made to Public Code as a pre-payment of the last 4 terms of periodic gift dated 24-08-2020. The board of Public Code decided to use that money to repay the loan (€ 1.702.000) + interest (€ 9.652) to Dia Holding on November 1st. This brings an advantage of € 118.013 for Public Code.

Reviewer: ```suggestion
In October 2020 a decision was made to change the structure of the gift. On November 1, a transfer of € 1.726.000 was made to the Foundation for Public Code as a pre-payment of the last 4 terms of periodic gift dated 24-08-2020. The board of the Foundation for Public Code decided to use that money to repay the loan (€ 1.702.000) + interest (€ 9.652) to Dia Holding on November 1st. This brings an advantage of € 118.013 for the Foundation for Public Code.
```

## General

> * The opening of a USD account is in progress and will help facilitate money transfers from KBFUS and to balance out the costs incurred by conversion rates and fees intermediary banks charge for international wire transfers.

Reviewer: ```suggestion
* The opening of a USD account is in progress and will help facilitate [money transfers from KBFUS](../../activities/philanthropic-fundraising/donating-united-states.md) and to balance out the costs incurred by conversion rates and fees intermediary banks charge for international wire transfers.
```

* 2FTE added each for 5500 gross per month. Amortized over the year. Estimated starting date March for 1FTE and May for the other 1FTE. The need to hire more staff depends on the results of successful membership development
* Salaries also include raises for some of the current staff
* Office cost from Q2 for 6000 per month
* Some budget for events and community from Q2
* Marketing, research and design costs including illustration work by Floris Deerenberg and the design work by Ryan Conlan, of which the latter can be covered for the most part by the gift from Josh Elkes.
* Development of a Jitsi Community user interface is included in the budget under this ledger as well. This interface is needed to enable our community to sign themselves up for our Jitsi Community instance.

## Interim conclusion

> The new funding loan came in time for the continuation of funding for the Foundation. The runway is extended; the Opex (operational expenses) can be funded for the upcoming 19 months. Even though the runway is extended by AS, there is a need to get inflow of cash by:

Reviewer: The reference to AS isn't self-explanatory. Would replacing it with a full name or a reference to a specific donation/loan make this more clear to a new reader?

Author: It would. It will differ from the original but any future original I will now make will take into account that it's written for to be out in the open and thus leaving abbreviations out of it

> * Contributions (members) - Provincie Zuid-Holland is our first member, but more is needed

Reviewer: Verb tense agreement.

```suggestion
* Contributions (members) - Provincie Zuid-Holland is our first member, but more are needed
```
